Antidote/CallsManager.m

```diff
@@ -222,6 +222,23 @@ - (void)updateCurrentCallInterface:(OCTCall *)call
         else {
             activeVC.callDuration = call.callDuration;
         }
+
+        if (call.friendSendingVideo && ! activeVC.videoView) {
+            AALogVerbose(@"provide video feed to video view");
+            activeVC.videoView = [self.manager videoFeed];
+        }
+        else if (! call.friendSendingVideo && ! call.videoIsEnabled) {
```

Comment:
Sorry I missed this. `friendSendingVideo` and `videoIsEnabled` is both needed to know if we need to update the UI for outgoing video versus just incoming video `friendSendingVideo`.